metrics/write.go

```diff
@@ -103,6 +103,7 @@ func (c *Client) write() error {
 	)
 
 	log.Printf("Sending:  %v timeseries, %v samples, %v timeseries per request, %v delay between requests\n", len(tss), c.config.RequestCount, c.config.BatchSize, c.config.RequestInterval)
+	ticker := time.Tick(c.config.RequestInterval)
```

Comment:
Would you be willing to use `time.NewTicker` here instead to avoid potential leaks?

Though not a big deal right now, it is possible that this function will be run in a loop in the future (for continuous testing).

Reply:
Sure, makes sense.